core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -140,7 +140,7 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
       // SPARK-28843: limit the OpenMP thread pool to the number of cores assigned to this executor
       // this avoids high memory consumption with pandas/numpy because of a large OpenMP thread pool
```

Comment:
comment should be updated if this goes in

Reply:
done